tfjs-layers/src/engine/container.ts

```diff
@@ -642,6 +652,10 @@ export abstract class Container extends Layer {
     batchSetValue(weightValueTuples);
   }
 
+  private isKerasKeras(str: string): boolean {
```

Comment:
We could add some comments to describe what's `keras-keras format`, like: `keras-keras format names weight as 'layerName/index', instead of layerName/weightName'`.